identifiers/country-pe/README.md

```diff
@@ -0,0 +1,7 @@
+# OCD Division Identifier
+
+This folder lists the following administrative divisions of Peru: 
+* Regions of Peru as **region**.
+* Provinces as **province**.
+
+The OCD identifiers are created based on the HASC code (Used sources [PE regions](http://www.statoids.com/upe.html))
```

Comment:
Is there a specific reason to use HASC code instead of ISO?

Reply:
No, in previous PR, we used to base OCD ids on one of them. Also, in PE case, HASC code is shorter than ISO.

Reply:
Got it. On one hand I don't think length matters much for OCD IDs. However, I think the guidelines basically just require consistency within a country. So HASC is probably fine.

@sguenther85 @jpmckinney any opinions on this matter?

Reply:
It's 2 characters vs 3 characters: https://en.wikipedia.org/wiki/ISO_3166-2:PE

Let's prioritize consistency, so please change to ISO.

Reply:
Done